suite/tests/CMakeLists.txt

```diff
@@ -2549,6 +2551,13 @@ endif ()
     tobuild_ci(client.destructor client-interface/destructor.cpp "" "" "")
   endif ()
 
+  if (UNIX)
```

Comment:
Repeating: comment as to why this is UNIX only (ref i# ideally).  I'm not sure myself: the asm is cross-platform now.

Reply:
I enabled it, but I could not test this on Windows, I don't have a AVX-512 capable machine with Windows. It actually found me a bug, it needs to be ${proc_supports_avx512}). I have added this fix and mentioned this in the commit message.